db/migrate/20210406003421_create_notes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateNotes < ActiveRecord::Migration
+  def change
+    create_table :notes do |t|
+      t.string :title
+      t.text :content
+      t.references :notable, polymorphic: true
```

Comment:
i usually find it a good idea to prevent null in the database where it makes sense to do so.

```suggestion
      t.string :title, null: false, default: ""
      t.text :content, null: false, default: ""
      t.references :notable, polymorphic: true, null: false
```

Reply:
Interesting.. I get a mysql2 error when attempting to migrate.
```
BLOB, TEXT, GEOMETRY or JSON column 'content' can't have a default value
```
According to [this](https://stackoverflow.com/questions/3466872/why-cant-a-text-column-have-a-default-value-in-mysql) it has got to do with how mysql stores `text` values. 
I've set the default value for the other, but left out content for now.